recipe/meta.yaml

```diff
@@ -10,6 +10,7 @@ source:
   sha256: f393389a0c22664613075cd22e784aee264e7ea4c4d8aa2accf2b2896c826150
 
 build:
+  noarch: python
```

Comment:
You will have to bump the build number, re-render, and always send the PR from a branch. If the CIs passed here a package would be uploaded anyway and it could be broken. More on how to maintain a feedstock can be seen here: https://conda-forge.org/docs/maintainer/00_intro.html